MVVMKit/View Controllers/MVVMCollectionViewController.swift

```diff
@@ -27,31 +27,57 @@ import UIKit
 /**
  A convenience class for a view controller handling a UICollectionView.
  The class fully implements the UICollectionViewDataSource.
+ 
+ `MVVMCollectionViewController` implements the following behaviors:
+ - It sets the data source of the collection view to self.
+ - It implements the viewWillAppear(_:) method and automatically calls `bind(viewModel:)` on first appearance.
+ - It implements the viewDidAppear(_:) method and automatically flashes the collection view's scroll indicators when it first appears.
  */
 open class MVVMCollectionViewController<Model: CollectionViewViewModel>: UIViewController, CollectionViewViewModelOwner, UICollectionViewDataSource {
+    public typealias CustomViewModel = Model
     
     @IBOutlet public weak var collectionView: UICollectionView! {
         didSet { collectionView.dataSource = self }
     }
     
-    public typealias CustomViewModel = Model
-    
-    /// Override this method to bind your view model to the view
-    open func bind(viewModel: Model) {
-        
-    }
-    
     /**
      The view controller view model
      */
     open var viewModel: Model? {
-        didSet { viewModel?.binder = self }
+        didSet {
+            viewModel?.binder = self
+            bindIfViewLoaded()
```

Comment:
Don't sure It make sense to add automatic binding with `DelegatingViewModel`.
Using the delegate pattern is the view model is in charge to call `bind` every time it needs it (to trigger a ui update).
Using the diffable version instead bind have a different meaning: it should be called **once** to bind publishers from the view model to the view. In this case I think automatic binding could have sense.
What do you think?